bundles/pixi.js-node/src/adapter/adapter.ts

```diff
@@ -19,45 +19,50 @@ export const NodeAdapter = {
     getWebGLRenderingContext: () => gl as unknown as typeof WebGLRenderingContext,
     /** Returns the fake user agent string of `node` */
     getNavigator: () => ({ userAgent: 'node' }),
-    /** Returns an empty base url */
-    getBaseUrl: () => ('/'),
+    /** Returns the path from which the process is being run */
+    getBaseUrl: () => process.cwd(),
+    /** Returns the root of the path from which the process is being run */
+    getRootUrl: () =>
+    {
+        if (process.platform === 'win32')
+        {
+            return `${process.cwd().split('\\')[0]}\\`;
+        }
+
+        return '';
+    },
     fetch: (url: RequestInfo, options?: RequestInit) =>
     {
         const request = new Request(url, options);
 
-        // if url is not absolute path then use fs to attempt to read the url
-        try
+        // check if urls starts with http(s) as only these are supported by node-fetch
+        if ((/^http(s)*:/).test(request.url))
```

Comment:
RegExp could be a little cleaner.
```suggestion
        if ((/^https?:/).test(request.url))
```